UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Changeling/ChangelingDNA.cs

```diff
@@ -0,0 +1,63 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Systems.Character;
+using UnityEngine;
+using Util;
+
+namespace Changeling
+{
+	public class ChangelingDNA : ICloneable
+	{
+		public int DnaID;
+		public string PlayerName = "";
+		public string Objectives;
+		public CharacterSheet CharacterSheet;
+		public JobType Job;
+		public List<string> BodyClothesPrefabID = new();
+
+		public void FormDNA(PlayerScript playerDataForDNA)
+		{
+			foreach (var clothe in playerDataForDNA.Mind.Body.playerSprites.clothes)
+			{
+				if (clothe.Value.GameObjectReference != null)
+				{
+					BodyClothesPrefabID.Add(clothe.Value.GameObjectReference.GetComponent<PrefabTracker>().ForeverID);
+				}
+			}
+
+			PlayerName = playerDataForDNA.playerName;
+			DnaID = playerDataForDNA.Mind.bodyMobID;
+			CharacterSheet = (CharacterSheet)playerDataForDNA.Mind.CurrentCharacterSettings.Clone();
+			try
+			{
+				Job = playerDataForDNA.PlayerInfo.Job;
+			}
+			catch
+			{
+				Job = JobType.ASSISTANT;
+				Logger.LogError("When creating DNA can`t find target job", Category.Changeling);
```

Comment:
[Missing caller info](https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/#0-in-code-documentation-and-clarity)

![image](https://github.com/unitystation/unitystation/assets/34368774/41047c94-2b11-49b8-93e2-4d32ac2b2d0d)